ibm/service/kubernetes/data_source_ibm_container_vpc_worker_pool.go

```diff
@@ -57,6 +58,39 @@ func DataSourceIBMContainerVpcClusterWorkerPool() *schema.Resource {
 				Computed:    true,
 				Description: "The operating system of the workers in the worker pool",
 			},
+			"secondary_storage": {
+				Type:        schema.TypeList,
+				Computed:    true,
+				Description: "The optional secondary storage configuration of the workers in the worker pool.",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"name": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"count": {
+							Type:     schema.TypeString,
```

Comment:
Should this be `schema.TypeInt` ? Or is it string to match the underlying API?

Reply:
yes, count and size should be int